pyincore/analyses/epnfunctionality/epnfunctionalityutil.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2019 University of Illinois and others. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License v2.0 which accompanies this distribution,
+# and is available at https://www.mozilla.org/en-US/MPL/2.0/
+
+import pandas as pd
+import networkx as nx
+
+
+class EpnFunctionalityUtil:
+
+    @staticmethod
+    def get_bad_edges(G, nodestate, linkstate=None, scol='s0'):
+        badnodes = nodestate.loc[nodestate.loc[:, scol] == 0, 'nodenwid'].values
```

Comment:
same here, just string case sensitivity check this line and line 17. In fact, we have to check this for other method that we missed like network related one, but I believe that we had no problem so far. So it might be safe but maybe we might want to check. It is up to you. You can ignore this comment

Reply:
I think this is our schema?